### ISSUE TITLE
how to see MB labels while training ?

### ISSUE THREAD

**Issue by 3koozy** [ContinualAI/avalanche#1088]
Hello to you all,
I am trying to use contrastive functions with avalanche and it's very important to include diverse classes in each minibatch as well as multiple samples per class (for similarity and dissimilarity between classes in latent space).

I have configured the strategy to use class-balanced data loaders and tried to use a mini-batch size larger than num_classes.

but is there any way to see the exact samples or the labels of each mini-batch while they are introduced to my model?

Thanks , Abdulaziz Alakooz.

**Comment by 3koozy** [comment]
can someone help me please

**Comment by vlomonaco** [comment]
Hi @3koozy!

> but is there any way to see the exact samples or the labels of each mini-batch while they are introduced to my model?

by default mini-batches in Avalanche have the form <x, y, ..., t> with arbitrary additional tensors between y and t. See the example here: https://github.com/ContinualAI/avalanche/blob/master/examples/dataloader.py#L40-L48
